lib/rust/parser/debug/tests/parse.rs

```diff
@@ -48,7 +48,7 @@ macro_rules! test {
 
 #[test]
 fn nothing() {
-    test("", block![]);
+    test("", block![()]);
```

Comment:
Does this mean that each EOF is a new line?

Reply:
This is just a change in the answer to "does an empty file contain a single empty line, or no lines".